shared/src/main/kotlin/com/egm/stellio/shared/web/ExceptionHandler.kt

```diff
@@ -0,0 +1,48 @@
+package com.egm.stellio.shared.web
+
+import com.egm.stellio.shared.model.*
+import com.egm.stellio.shared.util.ApiUtils
+import com.fasterxml.jackson.core.JsonParseException
+import com.github.jsonldjava.core.JsonLdError
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.ExceptionHandler
+import org.springframework.web.bind.annotation.RestControllerAdvice
+
+@RestControllerAdvice
+class ExceptionHandler {
+
+    @ExceptionHandler
+    fun transformErrorResponse(throwable: Throwable): ResponseEntity<String> =
+        when (throwable) {
+            is AlreadyExistsException ->
+                ResponseEntity.status(HttpStatus.CONFLICT)
```

Comment:
Maybe do a private method that takes status/ErrorResponse to avoid all the duplicate code